packages/react-tabs/src/common/_constants.scss

```diff
@@ -0,0 +1,14 @@
+$fontWeightLight: 100;
+$fontWeightSemilight: 300;
+$fontWeightRegular: 400;
+$fontWeightSemibold: 600;
+$fontWeightBold: 700;
+
+$animationEaseFunction1: cubic-bezier(0.1, 0.9, 0.2, 1.0);
+$animationEaseFunction2: cubic-bezier(0.1, 0.25, 0.75, 0.9);
+$animationDuration1: 0.167s;
+$animationDuration2: 0.267s;
+$animationDuration3: 0.367s;
+$animationDuration4: 0.467s;
+
+$CONTEXTUAL_MENU_ITEM_HEIGHT: 36px; // Must match CONTEXTUAL_MENU_ITEM_HEIGHT in ContextualMenu.cnstyles.ts
```

Comment:
I had to copy a bunch of _*.scss files from react-next/src/common into react-tabs/src/common. I'm not a fan of copying code, but I didn't see another established way to handle it for now. Any ideas? (cc @dzearing, @xugao)

Reply:
other than having a shared package to put these shared styles ('@fluentui/common-styles`?), i don't see other options. I think i am okay with tracking this as a TODO for now until another package needs these styles

Reply:
Well, this is already a thing in `react-button` and possibly `react-next` so I think we'd better think about a solution sooner rather than later.

Reply:
I didn't realize until now how much styles we are duplicating. We should create that shared package now instead of postponing :)

Reply:
Logged #14143 to track putting them in a common package. I'll get to it next, but I'd prefer not to delay this PR for it, if possible.